proto/one/common/core/content.proto

```diff
@@ -73,14 +73,15 @@ import "report_run.proto";
 import "ingest_client.proto";
 import "cellvalue_backup.proto";
 import "outputcell_backup.proto";
+import "operation.proto";
```

Comment:
Since you changed this to operation_export I don't think this import is going to work and will prevent building the protobufs